clang/lib/Sema/SemaConcept.cpp

```diff
@@ -483,6 +483,13 @@ bool Sema::CheckConstraintSatisfaction(
         *this, nullptr, ConstraintExprs, ConvertedConstraints,
         TemplateArgsLists, TemplateIDRange, OutSatisfaction);
   }
+  // Invalid templates could make their way here. Substituting them could result
+  // in dependent expressions.
+  // FIXME: Say something in the diagnostic note?
```

Comment:
I don't think we need that fixme. It would be weird to say "note: because foo is invalid" - we usually don;t do that and it's not useful unless we can explain _why_ it is invalid

Reply:
Removed